lib/process.ml

```diff
@@ -67,7 +70,9 @@ let add_link t (pid : Pid.t) = Atomic.set t.links (pid :: Atomic.get t.links)
 let add_signal t (signal : signal) =
   Atomic.set t.signals (signal :: Atomic.get t.signals)
 
-let send_message t msg = Mailbox.queue t.mailbox msg
+let send_message t msg =
+  Mailbox.queue t.mailbox msg;
+  mark_as_runnable t
```

Comment:
Moved this bit of logic into the Process module because i forgot twice to mark a process as runnable after delivering a message to it.

In the long run, the process state machine should move into this module as much as we can.